### PR TITLE
Snapshot tests for BPKCalendar

### PR DESCRIPTION
**Pull request by Szabadember**

### CODE REVIEW

File: Example/SnapshotTests/BPKCalendarSnapshotTest.m [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    
+    [self configureParentView:parentView forCalendar:bpkCalendar];
+    bpkCalendar.selectionType = BPKCalendarSelectionSingle;
+    bpkCalendar.selectedDates = @[ self.date1 ];
+    [bpkCalendar reloadData];
+    
+    FBSnapshotVerifyView(parentView, nil);
+}
+
+- (void)testCalendarWithRangeSelection {
```
Comment (tevelee) [inline Example/SnapshotTests/BPKCalendarSnapshotTest.m]: can you add another case for same-day range selection as well?

File: Example/SnapshotTests/BPKCalendarSnapshotTest.m [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    UIView *parentView = [[UIView alloc] initWithFrame:CGRectZero];
+    BPKCalendar *bpkCalendar = [[BPKCalendar alloc] initWithFrame:CGRectZero];
```
Comment (tevelee) [inline Example/SnapshotTests/BPKCalendarSnapshotTest.m]: Parent view and calendar init logic are there in all of the test cases. The same 2 lines of code. Maybe it's worth extracting them

**Comment by shaundon** [comment]
It all looks good, but I'm a bit confused by the year pill looking weird in the snapshots:

screen shot 2018-12-13 at 13 52 55

Do you know why this is, and is it a problem?

**Comment by shaundon** [comment]
Alright cool, sounds like you've mitigated it well :)

**Pull request merged**